src/core/bridge/speechsynthesizerbridge.h

```diff
@@ -15,7 +15,10 @@ extern "C" {
 #endif
 
 bool SpeechSynthesizerInit(void);
-void SpeechSynthesizerSpeak(const char* text);
+void SpeechSynthesizerUninitialize(void);
+
+void SpeechSynthesizerSpeakEnglish(const char* text);
```

Comment:
I'm not sure this function is needed. Do we really want to set up a different function for each language? We might as well use the more general function that already takes in a language code.

Do you think you need different code to be run per each language?

Reply:
Then in that case the preference should be a default parameter defined in the header.

Reply:
You're right that C doesn't support it. C++ does and I was confusing it for code in the class.
So let's look at this in perspective. This function is part of a public API in the bridge headers. Is a function for English only something we want to continue to support in 5 years? I think not. I think that also supersedes convenience on not passing in the language.

That doesn't mean we can't still use a "default" value. What about setting a CVar and/or a context variable on the speed synthesizer itself? You can set the context only once and then all calls will use that language.

Reply:
I'm happy with just removing it, but I really do not see the issue with having a context variable on the actual synthesizer class and a new method to change the context.